configure.ac

```diff
@@ -730,6 +736,15 @@ AS_IF([test "x$use_gperftools" != "xno"], [
     PKG_CHECK_MODULES([gperftools], [libprofiler, libtcmalloc])
 ])
 
+dnl Check for sanitizers
+if test "$use_sanitizer" = no
+then
+    sanitizer=""
+else
+    sanitizer="-fsanitize=$use_sanitizer"
+fi
+AC_SUBST(sanitizer)
```

Comment:
Cosmetic only, but maybe write this in fewer lines using this configure.ac pattern ...

```
sanitizer=""
if test "$use_sanitizer" = no; then
   sanitizer="-fsanitize=$use_sanitizer"
fi
AC_SUBST(sanitizer)
```

Reply:
done. thanks!